src/braket/ocean_plugin/braket_dwave_sampler.py

```diff
@@ -34,22 +34,22 @@ class BraketDWaveSampler(BraketSampler):
     A class for using DWave-formatted parameters and properties with Amazon Braket as a sampler.
 
     Args:
```

Comment:
It's weird to me that this is in the class doc string and not in the __init__ doc string.. Maybe this is a convention I'm not aware of